python/ray/autoscaler/gcp/example-truly-minimal.yaml

```diff
@@ -0,0 +1,7 @@
+# A unique identifier for the head node and workers of this cluster.
+cluster_name: minimal
+
+# Cloud-provider specific configuration.
+provider:
```

Comment:
Why are these different? can we reunify them (in particular, by improving the defaults in `defaults.yaml`?

Reply:
Sorry, I didn't mean to commit this extra file.